integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -333,7 +333,7 @@ const stubCreateBarcodeFailure = (prisonerName?: string): SuperAgentRequest => {
   })
 }
 
-const stubCreateContact = (): SuperAgentRequest =>
+const stubCreateContact = (contactId: number): SuperAgentRequest =>
```

Comment:
I was going to ask this, but the penny has dropped! 😁 

I _was_ going to ask why we've changed the create contact endpoint to take the contact ID, and to question how that would work because we wouldn't know the contact ID at that point, but then the penny dropped and I realised that this was the stub and that the contactID arg is for building the response! 👍  

Doh! 😁